### DISCUSSION TITLE
Geospatial Data?

### DISCUSSION THREAD

**Discussion by davidhmays** [cozodb/cozo#191] [Q&A]
Would CozoDB support whatever is needed to compute distances between address in geospatial data?
For example: finding the 10 closest address to a particular address in your database?

I know EdgeDB is supporting geospatial data now, but I wasn't sure how it is stored any differently, or if vectors could also be used to represent not just semantic closeness, but physical closeness, possibly a conversion to a longitude and lattitude vector?

**Comment by skyrod-vactai** [comment]
I was looking into this too - I believe it is possible to calculate now, since all the primitive functions needed are there. I don't really know the formula however. This might help: https://www.themathdoctors.org/distances-on-earth-1-the-cosine-formula/ and there are other formulas out there, if you google it. You just need to figure out how to specify it in the query. 

I assume by "address" you mean lat/lon coordinates, and not a street address.
